CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased Changes
 
+* Added component name to property validation error message
```

Comment:
```suggestion
* Added component name to property validation error message ([#275](https://github.com/Roblox/roact/pull/275))
```
Linking back to the PR containing a change is something we forget to do a lot, but is super useful when looking back over these!